### PR TITLE
[dev] AB#27857 - Identify Inputs gap between Sketch Cloud and cm-ui library

### PR DESCRIPTION
**Pull request by morethanfire**

### CODE REVIEW

File: docs/src/app/navigation.jsx [hunk 1]
```diff
@@ -364,192 +364,204 @@ class CoreAppNavigation extends React.PureComponent {
...
 
                             <li>
                                 <Link
                                     className={navItemClassName, classes.ulLink}
-                                    to={{ pathname: '/molecules/date-picker-calendar' }}
+                                    to={{ pathname: '/inputs/segmented-controls' }}
                                     activeClassName={isActive}
                                 >
-                                    Date Picker Calendar
+                                    Segmented Controls
```
[the remark this answers is not in the document]
Comment (morethanfire) [reply docs/src/app/navigation.jsx:490]: Yep, they should be asc. Thanks for the find.

File: src/inputs/datePickerDeprecatedCalendar.jsx [hunk 2]
```diff
@@ -1,37 +1,56 @@
...
 import Icon from '../dataDisplay/icon';
 
-import DatePickerUtils from '../utils/datePickerUtils.js';
+const propTypes = {
+    buttonClear: PropTypes.bool,
+    date: PropTypes.object, // Single date moment object. Coverted from timestamp in parent component.
+    dateEnd: PropTypes.object, // Range end date moment object. Coverted from timestamp in parent component.
+    dateSecondaryEnd: PropTypes.object, // Range start date moment object. Coverted from timestamp in parent component.
+    dateSecondaryStart: PropTypes.object, // Range start date moment object. Coverted from timestamp in parent component.
+    dateStart: PropTypes.object, // Range start date moment object. Coverted from timestamp in parent component.
```
[the remark this answers is not in the document]
Comment (morethanfire) [reply src/inputs/datePickerDeprecatedCalendar.jsx:21]: I change theme to `shape({})`

File: src/inputs/dropdownItem.jsx [hunk 3]
```diff
@@ -31,4 +16,17 @@ DropdownItem.propTypes = {
     style: PropTypes.object,
 };
 
+function DropdownItem(props) {
+    const { className, style } = props;
+    const containerClasses = ClassNames('ui', 'dropdown', className);
+
+    return (
+        <div className={containerClasses} style={style}>
+            asdf
```
[the remark this answers is not in the document]
Comment (morethanfire) [reply src/inputs/dropdownItem.jsx:25]: No, it's kind of a dummy file. :/

File: src/inputs/datePickerDeprecatedInput.jsx [hunk 4]
```diff
@@ -119,7 +119,7 @@ class DatePickerInput extends React.Component {
         });
     }
 
-    onChange(dateType, value) {
+    _onChange(dateType, value) {
```
Comment (groberts314) [inline src/inputs/datePickerDeprecatedInput.jsx:122]: Was this intentional?

File: src/inputs/datePickerDeprecatedCalendar.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (groberts314) [reply src/inputs/datePickerDeprecatedCalendar.jsx]: Shape _all_ the things! =)

File: src/inputs/datePickerDeprecatedInput.jsx (hunk 4 above)
Comment (morethanfire) [reply src/inputs/datePickerDeprecatedInput.jsx]: Nope, thanks

File: src/inputs/dropdownItem.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (groberts314) [reply src/inputs/dropdownItem.jsx:25]: Yeah, it's been like this forever I think.  We can continue not worrying about it. :-)

[the remark this answers is not in the document]
Comment (morethanfire) [reply src/inputs/dropdownItem.jsx:25]: Yeah, unfortunately, but soon we can worry about it.

**Pull request merged**